tests/testthat/test_MsBackendDataFrame.R

```diff
@@ -321,13 +505,23 @@ test_that("spectraData, spectraData<-, MsBackendDataFrame works", {
     res <- spectraData(be, c("mz"))
     expect_true(is(res, "DataFrame"))
     expect_equal(colnames(res), "mz")
-    expect_equal(res$mz, SimpleList(numeric(), numeric()))
+    expect_equal(res$mz, NumericList(numeric(), numeric(), compress = FALSE))
 
     res <- spectraData(be, c("a", "intensity"))
     expect_true(is(res, "DataFrame"))
     expect_equal(colnames(res), c("a", "intensity"))
-    expect_equal(res$intensity, SimpleList(numeric(), numeric()))
+    expect_equal(res$intensity, NumericList(numeric(), numeric(),
+                                            compress = FALSE))
     expect_equal(res$a, c("a", "a"))
+
+    spectraData(be) <- DataFrame(fromFile = 1L, mzLevel = c(3L, 4L),
+                                 rtime = c(1.2, 1.4), other_col = "b")
+    expect_identical(rtime(be), c(1.2, 1.4))
+    expect_true(any(spectraVariables(be) == "other_col"))
+    expect_identical(spectraData(be, "other_col")[, 1], c("b", "b"))
+
+    expect_error(spectraData(be) <- DataFrame(msLevel = 1:3, fromFile = 1),
+                 "with 2 rows")
 })
 
 test_that("show,MsBackendDataFrame works", {
```

Comment:
This test is empty, did you want to use `expect_output` or similar here?

Reply:
I guess so, just forgot. But the `Spectra` will get a  major update anyway, so I would let it for now as it is.